vyper/builtins/functions.py

```diff
@@ -296,20 +296,25 @@ class Slice(BuiltinFunction):
 
     def evaluate(self, node):
         (lit, st, le) = node.args[:3]
```

Comment:
is it really needed to abbreviate this? It makes much easier to read with full names.

Reply:
fair enough!

Reply:
agreed, also i think unwrapping `node.args[:3]` removes a bit of safety. i'd prefer `literal_value, start, length = node.args`

Reply:
Thanks, yeah, renamed and tupled!